english/you-are-in-minecraft/gather_wood/gather_wood.md

```diff
@@ -0,0 +1,11 @@
+You look at the nearby forest and you see birch trees. Your minecraft instinct kicks in and your mind says 'MUST COLLECT THEM'. You notice your food level going down and you realize you are in survival mode and you need to survive.
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.